.github/workflows/branches-and-prs.yml

```diff
@@ -36,18 +36,11 @@ jobs:
         with:
           # Codecov needs fetch-depth > 1
           fetch-depth: 2
-      - name: Cache Gradle wrapper and dependencies
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.gradle/caches/
-            ~/.gradle/wrapper/dists
-          key: test-${{ runner.os }}-variant-${{ matrix.variant }}-gradle-${{ hashFiles('**/*.gradle', '**/*.gradle.kts', '**/gradle.properties', 'gradle/**', 'buildSrc/src/main/**') }}
-          restore-keys: |
-            test-${{ runner.os }}-variant-${{ matrix.variant }}-
       - name: 'Set up JDK'
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: 'adopt'
```

Comment:
Maybe switch to Temurin now that `adopt` won't publish any updates anymore?

Reply:
yeah, but I didn't want to mix two things, AFAIK `adopt` was the default in v1.